Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/CounterFactory.cs

```diff
@@ -0,0 +1,615 @@
+                        "gen1HeapSize",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\Gen 2 heap size":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "gen2HeapSize",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\Large Object Heap size":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "largeObjectHeapSize",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\Finalization Survivors":
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# GC Handles":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "gcHandles",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\Allocated Bytes/ sec":
+                    return new RateCounterGauge(
+                        reportAs,
+                        "allocatedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Induced GC":
+                    return new RateCounterGauge(
+                        reportAs,
+                        "inducedGC",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Bytes in all Heaps":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "bytesInAllHeaps",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Total committed Bytes":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "commitedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Total reserved Bytes":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "reservedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# of Pinned Objects":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "pinnedObjects",
+                        AzureWebApEnvironmentVariables.CLR);
+                
+                //// Quick pulse related hard coded performance counters.
+                case @"\Memory\Committed Bytes":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "committedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\Processor(_Total)\% Processor Time":
```

Comment:
It is a stratch to map Processor CPU to the single process CPU. From my standpoint it is a wrong mapping.